.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,137 @@
+          create user "${{ env.OPENBOXES_USER }}"@"localhost" identified by "${{ env.OPENBOXES_PASSWORD }}";
+          grant all privileges on ${{ env.DB_DATABASE }}.* to "${{ env.OPENBOXES_USER }}"@"localhost";
+          flush privileges;'
+
+    - name: 'load ~/.grails cache'
+      uses: actions/cache@v3
+      with:
+        key: ${{ runner.os }}-v3-grails-${{ hashFiles('**/grails-app/conf/BuildConfig.groovy') }}
+        path: ~/.grails
+
+    - id: grails-sdk
+      name: 'load ~/grails-1.3.9 cache'
+      uses: actions/cache@v3
+      with:
+        key: ${{ runner.os }}-v3-grails-1.3.9-${{ hashFiles('**/grails-app/conf/BuildConfig.groovy') }}
+        path: ~/grails-1.3.9
+
+    - name: 'load ~/.ivy2/ cache'
+      uses: actions/cache@v3
+      with:
+        key: ${{ runner.os }}-v3-ivy2-${{ hashFiles('**/grails-app/conf/BuildConfig.groovy') }}
+        path: ~/.ivy2
+
+    - name: 'load ${{ ''$GITHUB_WORKSPACE'' }}/node_modules cache'
+      uses: actions/cache@v3
+      with:
+        key: ${{ runner.os }}-v3-node_modules-${{ hashFiles('**/package-lock.json') }}
+        path: ${{ github.workspace }}/node_modules
+
+    - name: 'load ~/.npm cache'
+      uses: actions/cache@v3
+      with:
+        key: ${{ runner.os }}-v3-npm-${{ hashFiles('**/package-lock.json') }}
+        path: ~/.npm
+
+    - id: sdkman
+      if: steps.grails-sdk.outputs.cache-hit != 'true'
+      name: download grails SDK (unless cached)
+      uses: sdkman/sdkman-action@master
+      with:
+        candidate: grails
+        version: 1.3.9
+
+    - if: steps.grails-sdk.outputs.cache-hit != 'true'
+      name: unzip grails SDK (unless cached)
+      run: unzip -o ${{ steps.sdkman.outputs.file }} -d ~/
+
+    - name: npm install
+      run: npm install
+
+    - name: npm rebuild node-sass
+      run: npm rebuild node-sass
+
+    - name: npm run bundle
+      run: npm run bundle
+
+    - name: configure OpenBoxes for database access and testing
+      run: |
+        mkdir -p ~/.grails
+        echo "dataSource.password=${{ env.OPENBOXES_PASSWORD }}" > ~/.grails/openboxes-config.properties
+        echo "dataSource.url=jdbc:mysql://localhost:3306/${{ env.DB_DATABASE }}?allowPublicKeyRetrieval=true&useSSL=false" >> ~/.grails/openboxes-config.properties
+        echo "dataSource.username=${{ env.OPENBOXES_USER }}" >> ~/.grails/openboxes-config.properties
+        echo "openboxes.fixtures.enabled=true" >> ~/.grails/openboxes-config.properties
+        echo "openboxes.jobs.dataCleaningJob.enabled=false" >> ~/.grails/openboxes-config.properties
+        echo "openboxes.jobs.refreshDemandDataJob.enabled=false" >> ~/.grails/openboxes-config.properties
+        echo "openboxes.jobs.refreshInventorySnapshotJob.enabled=false" >> ~/.grails/openboxes-config.properties
+        #
+        # FIXME Strangely, enabled = false doesn't disable refreshProductAvailabilityJob,
+        # FIXME but setting delayInMilliseconds to a high value has the same practical effect.
```

Comment:
^^^ I'm not sure why setting `enabled=false` doesn't work here; my uninformed guess is something something Docker character-set something something... the other changes work properly and the workaround is perfectly fine. Still, if either if you has insight into this, I'm curious and all ears!

Reply:
Doesn't look like this was resolved since we're still sending this to .properties file instead of .groovy.

Reply:
Resolved in ed446e3de.